src/persist-client/src/read.rs

```diff
@@ -547,10 +563,12 @@ where
     TFn: FnMut(&mut T) -> bool,
 {
     let mut retry = Retry::persist_defaults(SystemTime::now()).into_retry_stream();
+    let get_span = debug_span!("fetch_batch::get");
```

Comment:
do we also want a top-level span on `fetch_batch_part`?

Reply:
ditto similar logic as `apply_unbatched_cmd`. there's not much of note between here and the public API method that uses it. also ditto subject to change my opinion here as soon as we start using this for real debugging